src/Echidna.Tests/Mapping/DictionaryMappingTest.cs

```diff
@@ -74,9 +74,9 @@ public void TestCanConvertValues([Values] Db db)
     }
 
     [Test]
-    public void TestAttemptsToConvertAllValues()
+    public void TestAttemptsToConvertAllValues([ValueSource(typeof(TestHelper), nameof(TestHelper.DbsToTest))] Db db)
     {
-        using var reader = Db.SqlServer.Read("SELECT 1 AS a, 'b' as b");
```

Comment:
In some cases, the tests were running only against one database, so I've updated them to test against all databases.

Note that I didn't do this in some tests where this caused a test failure ([example](https://github.com/madelson/Echidna/pull/5/files#diff-f325e2591ef6494caf82fbd6f84ed20b2f60459680467efab01e7f1a2e3826e0R105)), but that's a problem for another day.